src/test/java/com/adobe/target/edge/client/ondevice/collator/PageParamsCollatorTest.java

```diff
@@ -44,13 +44,70 @@ public void testCollator() {
     assertEquals("/about/", result.get(PageParamsCollator.PAGE_PATH_LOWER));
     assertEquals("WWW.TARGET.ADOBE.COM", result.get(PageParamsCollator.PAGE_DOMAIN));
     assertEquals("www.target.adobe.com", result.get(PageParamsCollator.PAGE_DOMAIN_LOWER));
-    assertEquals("TARGET", result.get(PageParamsCollator.PAGE_SUBDOMAIN));
-    assertEquals("target", result.get(PageParamsCollator.PAGE_SUBDOMAIN_LOWER));
+    assertEquals("WWW.TARGET", result.get(PageParamsCollator.PAGE_SUBDOMAIN));
```

Comment:
www is technically a subdomain

Reply:
you should keep this test in tact and change the implementation so it works.  This it testing the functionality that is in place in production today (delivery API).

Reply:
fixed @jasonwaters